src/include/gurt/telemetry_common.h

```diff
@@ -10,13 +10,122 @@
 
 #define D_TM_VERSION			1
 #define D_TM_MAX_NAME_LEN		256
-#define D_TM_MAX_SHORT_LEN		64
-#define D_TM_MAX_LONG_LEN		1024
+#define D_TM_MAX_DESC_LEN		64
+#define D_TM_MAX_UNIT_LEN		16
 #define D_TM_TIME_BUFF_LEN		26
 
 #define D_TM_SHARED_MEMORY_KEY		0x10242048
 #define D_TM_SHARED_MEMORY_SIZE		(1024 * 1024)
 
+/**
+ * The following definitions are suggested strings for units that may be used
+ * when explicitly calling d_tm_add_metric() to initialize a metric before use.
+ */
+
+/** Units of measure for time */
+#define D_TM_YEAR			"y"
+#define D_TM_MONTH			"mo"
+#define D_TM_WEEK			"w"
+#define D_TM_DAY			"d"
+#define D_TM_HOUR			"h"
+#define D_TM_MINUTE			"min"
+#define D_TM_SECOND			"s"
+#define D_TM_MILLISECOND		"ms"
+#define D_TM_MICROSECOND		"us"
+#define D_TM_NANOSECOND			"ns"
```

Comment:
Stray tab? Or maybe it's just the GH UI.

Reply:
It renders correctly if I do a "view file" through GH.  It also renders correctly with VI and Visual Studio.  I'm not sure why the GH UI shows the erroneous tab.